doc/Scripts/script_notes.md

```diff
@@ -0,0 +1,37 @@
+# Scripts
+## launch_repairnator.sh
+The launch repairnator has two options:
+```
+./launch_repairnator.sh
```

Comment:
maybe, put the standard usage: `./launch_repairnator.sh [list_of_buildIDS.txt]`
In usages `[]` are expressing an optional argument